nemo/core/config/pytorch_lightning.py

```diff
@@ -35,7 +35,7 @@ class TrainerConfig:
         https://pytorch-lightning.readthedocs.io/en/latest/trainer.html#
     """
 
-    logger: Any = True
+    logger: Any = False
```

Comment:
@ericharper  This should be reverted, signature must match PTL Trainer exactly. It is NemoConfig which should override this value to False..

Reply:
Done.